### PR TITLE
initial backend

### PR DESCRIPTION
**Pull request by ghost**
WHAT: 
Initial base backend

### CODE REVIEW

**Review by DungHuuNguyen — APPROVED** [review]

**Comment by ghost** [comment]
ready

File: backend/routes/web.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+use Illuminate\Support\Facades\Route;
+use App\Http\Controllers\LanguageController;
+use App\Http\Middleware\Localization;
+Route::get('/', function () {
+    return view('welcome');
+});
+
+
+Route::get('language/{lang}', [LanguageController::class, 'changeLanguage'])->middleware(Localization::class);
```
Comment (binhpt-3177) [inline backend/routes/web.php]: endline

**Review by binhpt-3177 — APPROVED** [review]
Nhờ a Thịnh review phần BE

**Comment by ghost** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**